launch/move_base.launch

```diff
@@ -8,26 +8,24 @@
 
 	<node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen" clear_params="true" unless="$(arg with_mux)">
 		<!-- default:20.0. with this value dwa planner fails to find a valid plan a lot more -->
-                <rosparam file="$(find strands_movebase)/strands_movebase_params/move_base_params.yaml" command="load"/>
+        <rosparam file="$(find strands_movebase)/strands_movebase_params/move_base_params.yaml" command="load"/>
 		<rosparam file="$(find scitos_2d_navigation)/scitos_move_base_params/costmap_common_params.yaml" command="load" ns="global_costmap" />
 		<rosparam file="$(find scitos_2d_navigation)/scitos_move_base_params/costmap_common_params.yaml" command="load" ns="local_costmap" />
 		<rosparam file="$(find scitos_2d_navigation)/scitos_move_base_params/local_costmap_params.yaml" command="load" />
 		<rosparam file="$(find scitos_2d_navigation)/scitos_move_base_params/global_costmap_params.yaml" command="load" />
 		<rosparam file="$(find scitos_2d_navigation)/scitos_move_base_params/dwa_planner_ros.yaml" command="load" />
+		<rosparam file="$(find scitos_2d_navigation)/scitos_move_base_params/nav_fn_ros.yaml" command="load" />
 	</node>
 	<node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen" clear_params="true" if="$(arg with_mux)">
 		<!-- default:20.0. with this value dwa planner fails to find a valid plan a lot more -->
         <remap from="/cmd_vel" to="/cmd_vel_mux/input/navigation"/>
-		<param name="controller_frequency" value="10.0" />
-		<param name="controller_patience" value="15.0" />
-		<param name="planner_frequency" value="2.0" />
-		<param name="clearing_rotation_allowed" value="false" />
```

Comment:
Are all these not needed anymore? It seems unrelated to the "avoiding planning through unknown areas"?